frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+
+describe('Check noobaa link in obejct service dashboard and perform SSO', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/dashboards/object-service');
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Test admin user login for noobaa', () => {
+    //cy.visit('/dashboards/object-service');
+    cy.byLegacyTestID('system-name-mcg').eq(0)
```

Comment:
Let's update the UI to make sure `data-test-id` is not reused inside the same space.
```suggestion
    cy.byLegacyTestID('system-name-mcg')
```